public/css/admin.css

```diff
@@ -1,13 +1,20 @@
 /*--------------- MAILING LISTS -----------------*/
-h3 {
+
+head {
 	font-family: "Source Code Pro", monospace;
 }
 
+.mailing-address-list {
+	font-size: 14px;
+	color: var(--black);
+	background-color: var(--black);
+}
+
 #mailing-lists {
 	font-family: "Source Code Pro", monospace;
 	font-weight: bold;
 	font-size: 52px;
-	color: #50b5dd;
+	color: #619b83;
```

Comment:
Just for consistency's sake, please use CSS variables wherever possible (`var(--green)` here)